boards/arm/nucleo_f412zg/nucleo_f412zg.dts

```diff
@@ -19,15 +19,15 @@
 
 	leds {
 		compatible = "gpio-leds";
-		green_led_1: led@0 {
+		green_led_1: led_1 {
```

Comment:
The same here.

Reply:
Right. Script abuse is bad. Will fix. Thanks